static/css/main.css

```diff
@@ -31,3 +31,21 @@ ul {
   margin: 0;
   padding-left: 1em;
 }
+
+@media only screen and (max-width : 450px) {
+  .org {
+    flex: 100%;
+  }
+}
+
+@media only screen and (min-width : 450px) and (max-width : 600px) {
+  .org {
+    flex: 50%;
+  }
+}
+
+@media only screen and (min-width : 600px) and (max-width : 750px) {
```

Comment:
What happens if you remove the `min-width`, I think it should work fine ?

Reply:
no, it doesn't work